symfony/apache-pack/1.0/public/.htaccess

```diff
@@ -18,6 +18,10 @@ DirectoryIndex index.php
 </IfModule>
 
 <IfModule mod_rewrite.c>
+    # This Option needs to be enabled for RewriteRule, otherwise it will show an error like
+    # 'Options FollowSymLinks or SymLinksIfOwnerMatch is off which implies that RewriteRule directive is forbidden'
+    Options +FollowSymlinks
+
```

Comment:
This is added as a comment just a few lines above. 

I suggest to not change the defaults and let the user uncomment line 12 instead.

Reply:
@fabpot WDYT?

Reply:
Last time I used Apache was probably more than 15 years ago, so I have no idea.
I would tend to trust people using Apache :)

Reply:
Same here